tests/suites/test_suite_x509write.function

```diff
@@ -28,6 +32,74 @@ size_t mbedtls_rsa_key_len_func( void *ctx )
 }
 #endif /* MBEDTLS_RSA_C */
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+static int wrap_key_as_opaque( mbedtls_pk_context *pk, psa_algorithm_t hash_alg )
```

Comment:
This is replicated from #2183. As mentioned in https://github.com/ARMmbed/mbedtls/pull/2183/files#r234435165, I think we should introduce it as shared code in `pk_internal.h` in #2162. @mpg @AndrzejKurek Please let me know what you think.